### PR TITLE
Feat: enabling runner and diffrunner fuzz targets

### PR DESCRIPTION
**Pull request by andrei-near**
In this PR I enable the remain fuzzers: runner and diffrunner. Some additional changes done:

- Because runner fuzz binary depends on compiled contract wasm file, I changed test-near-contracts lib to load the wasm contract into binary at compile time instead of trying to read it from disk at runtime. Tested resulting binary on Nayduck nodes.
- Commented unused fuzz targets from fuzz.toml
- Because of above, I had to rename directories from fuzz_targets to fuzz_targets disabled so that buildkite sanity checks pass. This is no-op as these fuzzers are disabled

With this PR fuzz targets build for **master** branch [succeeds]((https://github.com/near/nearcore/actions/runs/4283565439/jobs/7459305461)) and ClusterFuzz jobs are fuzzing correctly the resulting targets.

### CODE REVIEW

File: runtime/near-test-contracts/src/lib.rs [hunk 1]
```diff
@@ -48,7 +48,11 @@ pub fn sized_contract(size: usize) -> Vec<u8> {
 /// on a specific protocol version, it should use [`base_rs_contract`].
 pub fn rs_contract() -> &'static [u8] {
     static CONTRACT: OnceCell<Vec<u8>> = OnceCell::new();
-    CONTRACT.get_or_init(|| read_contract("test_contract_rs.wasm")).as_slice()
+    let contract =
+        include_bytes!(concat!(env!("CARGO_MANIFEST_DIR"), "/res/", "test_contract_rs.wasm"))
+            .to_vec();
+
+    CONTRACT.get_or_init(|| contract).as_slice()
```
Comment (jakmeier) [inline runtime/near-test-contracts/src/lib.rs:55]: I think you can remove the once cell for lazy initialization. `include_bytes!` already gives you a slice with static lifetime, so you should be able to just return it.
```rust
pub fn rs_contract() -> &'static [u8] {
    include_bytes!(concat!(env!("CARGO_MANIFEST_DIR"), "/res/", "test_contract_rs.wasm"))
}
```
This also avoids copying into heap memory. (in your code `to_vec` allocates new memory)

**Review by jakmeier — COMMENTED** [review]
I am not familiar with the fuzzing infra, but I left a comment on the modified Rust code.

File: runtime/near-test-contracts/src/lib.rs (hunk 1 above)
Comment (andrei-near) [reply runtime/near-test-contracts/src/lib.rs:55]: Thanks for suggestion, applied.

**Review by jakmeier — APPROVED** [review]

**Comment by jakmeier** [comment]
I'm not familiar with the exact targets you enable with this PR and what the story behind `fuzz_targets_disabled` is, but the PR mostly looks good to me.

**Pull request merged**